ogr/ogrsf_frmts/wfs/ogroapifdriver.cpp

```diff
@@ -3265,6 +3280,8 @@ void RegisterOGROAPIF()
         "    <Value>AUTHORITY_COMPLIANT</Value>"
         "    <Value>GIS_FRIENDLY</Value>"
         "  </Option>"
+        "  <Option name='DATETIME' type='string' "
+        "description='Date-time filter to pass to items requests'/>"
```

Comment:
Maybe a few words longer "Date-time filter to pass to items requests with the datatime parameter". Times can also be used in queries in another way by using the filter and CQL2 extensions (once GDAL supports them).

Reply:
thanks for the suggestion. integrated